pages/table/simulated-server-actions.page.tsx

```diff
@@ -0,0 +1,157 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef, useState } from 'react';
+import { useCollection } from '@cloudscape-design/collection-hooks';
+import Button from '~components/button';
+import CollectionPreferences, { CollectionPreferencesProps } from '~components/collection-preferences';
+import Header from '~components/header';
+import Pagination from '~components/pagination';
+import Table from '~components/table';
+import TextFilter from '~components/text-filter';
+import { Instance, generateItems } from './generate-data';
+import {
+  columnsConfig,
+  EmptyState,
+  getMatchesCountText,
+  paginationLabels,
+  pageSizeOptions,
+  contentDisplayPreference,
+  defaultPreferences,
+} from './shared-configs';
+import { contentDisplayPreferenceI18nStrings } from '../common/i18n-strings';
+
+const allItems = generateItems();
```

Comment:
How come that linter did not complain about mixed imports and code declarations?